core/rawdb/accessors_indexes.go

```diff
@@ -129,3 +130,30 @@ func WriteBloomBits(db ethdb.KeyValueWriter, bit uint, section uint64, head comm
 		log.Crit("Failed to store bloom bits", "err", err)
 	}
 }
+
+// DeleteBloombits removes all compressed bloom bits vector belonging to the
+// given section range and bit index.
+func DeleteAllBloombits(db ethdb.Database, bit uint, from uint64, to uint64) {
```

Comment:
-All

Reply:
All in all I have similar comments here as with the hash iterator, can't we more preceisely iterate just what we need, instead of a larger range and trying to find the starting element, skipping everything before?